selenium/src/test/java/au/org/emii/portal/tests/step3/ShapeFileTest.java

```diff
@@ -0,0 +1,29 @@
+package au.org.emii.portal.tests.step3;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class ShapeFileTest extends BaseTest {
+
+    private static Logger log = Logger.getLogger(ShapeFileTest.class.getName());
+
+    @Test
+    public void verifyShapeFile() {
+        getDriver().get(AODN_PORTAL_SEARCH_PAGE + "?uuid=9d6b28d2-2d6b-4726-b603-ab03e3d15d34");
```

Comment:
Yeah there's no way around this for now. We can't find out via a selenium test which collections should have a shapefile, so we have to just pass it one manually.